contracts/token/ERC721/ERC721Basic.sol

```diff
@@ -1,27 +1,57 @@
 pragma solidity ^0.4.23;
 
+import "../../SupportsInterfaceWithLookup.sol";
+
 
 /**
  * @title ERC721 Non-Fungible Token Standard basic interface
  * @dev see https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md
  */
-contract ERC721Basic {
+contract ERC721Basic is SupportsInterfaceWithLookup {
   event Transfer(
     address indexed _from,
     address indexed _to,
-    uint256 _tokenId
+    uint256 indexed _tokenId
```

Comment:
do we have a policy for which ones to make indexed?

Reply:
This is actually looking at the [current spec](https://eips.ethereum.org/EIPS/eip-721) and making sure it's conformant.